### PR TITLE
[FIX] web, project, crm: allow hiding the properties option "View In Kanban"

### PR DESCRIPTION
**Pull request by std-odoo**
Purpose
=======
Some models might use the properties field in the kanban view, some other no. For that purpose, we want to add an option in order to hide the "View In Kanban" checkbox in the properties definition.

Task-3012780

### CODE REVIEW

File: addons/web/static/src/views/fields/properties/properties_field.js [hunk 1]
```diff
@@ -471,10 +472,12 @@ PropertiesField.components = {
 PropertiesField.props = {
     ...standardFieldProps,
     columns: { type: Number, optional: true },
+    hideKanbanOption: { type: Boolean, optional: true },
 };
 PropertiesField.extractProps = ({ attrs, field }) => {
     const columns = parseInt(attrs.columns || "1");
-    return { columns };
+    const hideKanbanOption = !!parseInt(attrs.hideKanbanOption || "0");
```
Comment (awa-odoo) [inline addons/web/static/src/views/fields/properties/properties_field.js:479]: I see there is a "archParseBoolean" util method, would probably be better than using a parseInt?

**Review by awa-odoo — COMMENTED** [review]
Hello @std-odoo 

One light comment, looking good :)

Other than that, was wondering if we should split the commits?
I usually like having the "/web" library change separated from the modules changes.

That will also allow having more precise commit messages.

Thanks & cheers!

**Comment by std-odoo** [comment]
@awa-odoo Changes done :)

**Comment by dbeguin** [comment]
@robodoo r+

**Comment by dbeguin** [comment]
@robodoo rebase-merge

**Review by sdegueldre — COMMENTED** [review]
Should probably introduce a test when introducing a feature to make sure it's not broken by subsequent refactorings, code looks fine :+1:

**Pull request closed without merging**